crates/autopilot/src/solvable_orders.rs

```diff
@@ -474,7 +475,8 @@ fn get_orders_with_native_prices(
     // and prices that have orders.
     let mut filtered_market_orders = 0_i64;
     let mut used_prices = BTreeMap::new();
-    let (usable, filtered): (Vec<_>, Vec<_>) = orders.into_iter().partition(|order| {
+    let mut tokens_by_priority = indexmap::IndexSet::new();
```

Comment:
Do we use this newly created index set anywhere (the logic to replace high priority tokens seem removed)?

Reply:
Good catch. This was intended to be used with `replace_high_priority()`.
I also tried to get too fancy when constructing the second indexset which turned out to be incorrect.
Now I just call `prioritize_missing_prices()` with the `filtered` orders again. (basically all as it was before)